saos-persistence/src/test/java/pl/edu/icm/saos/persistence/repository/JudgmentRepositoryTest.java

```diff
@@ -77,6 +76,69 @@ public void testSaveAndGet() {
         Assert.assertEquals(1, judgmentRepository.count());
     }
     
+    @Test
+    public void save_iterable() {
+        
+        // given
+        
+        CommonCourtJudgment judgment1 = new CommonCourtJudgment();
+        judgment1.addCourtCase(new CourtCase("AAA1"));
+        judgment1.getSourceInfo().setSourceCode(SourceCode.COMMON_COURT);
+        judgment1.getSourceInfo().setSourceJudgmentId("1");
+        
+        CommonCourtJudgment judgment2 = new CommonCourtJudgment();
+        judgment2.addCourtCase(new CourtCase("AAA2"));
+        judgment2.getSourceInfo().setSourceCode(SourceCode.COMMON_COURT);
+        judgment2.getSourceInfo().setSourceJudgmentId("2");
+        
+        DateTime beforeSave = new DateTime();
+        waitForTimeChange();
+        
+        
+        // execute
+        
+        judgmentRepository.save(Lists.newArrayList(judgment1, judgment2));
+        
+        
+        // assert
+        
+        assertEquals(2, judgmentRepository.count());
+        
+        Judgment retJudgment1 = judgmentRepository.findOne(judgment1.getId());
+        Judgment retJudgment2 = judgmentRepository.findOne(judgment2.getId());
+        
+        assertTrue(retJudgment1.getModificationDate().isAfter(beforeSave));
+        assertTrue(retJudgment2.getModificationDate().isAfter(beforeSave));
+    }
+    
+    @Test
+    public void saveAndFlush() {
```

Comment:
You don't check the change of the indexed flag nor indexed date.
You don't check the update of the judgment either.